fhir/com.b2international.snowowl.fhir.core/src/org/linuxforhealth/fhir/model/r5/type/code/FHIRTypes.java

```diff
@@ -0,0 +1,4836 @@
+/*
+ * (C) Copyright IBM Corp. 2019, 2022
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.linuxforhealth.fhir.model.r5.type.code;
+
+import org.linuxforhealth.fhir.model.annotation.System;
+import org.linuxforhealth.fhir.model.r5.type.Code;
+import org.linuxforhealth.fhir.model.r5.type.Extension;
+import org.linuxforhealth.fhir.model.r5.type.String;
+
+import java.util.Collection;
+import java.util.Objects;
+
+import javax.annotation.Generated;
+
+@System("http://hl7.org/fhir/fhir-types")
+@Generated("org.linuxforhealth.fhir.tools.CodeGenerator")
+public class FHIRTypes extends Code {
```

Comment:
Please remove the ones we do not support.

Reply:
Similar to `FHIRAllTypes`, a binding exists for this code system as well.

Reply:
🆗